app/code/Magento/Dhl/Model/Carrier.php

```diff
@@ -939,7 +940,7 @@ protected function _getDimension($dimension, $configWeightUnit = false)
             );
         }
 
-        return round($dimension, 3);
+        return $dimension ? round($dimension, 3) : 0.0;
```

Comment:
type casting might be more suitable here

Reply:
Hello @xmav 
Thank you for your review
Fixed, please check.